### DISCUSSION TITLE
When we can expect the 2.8.0 release?

### DISCUSSION THREAD

**Discussion by tanvir-retailai** [go-resty/resty#639] [General]
[https://github.com/go-resty/resty/issues/630](https://github.com/go-resty/resty/issues/630)  this issue had been fixed and set for v2.8.0 release. @jeevatkm are you going to release v2.8.0 soon ?

**Comment by jeevatkm** [comment]
@tanvir-retailai I will aim for sooner. However I noticed that PR introduced unexpected behavior for `io.Reader` flow. I will be fixing that before making a release.

**Reply by tanvir-retailai** [reply]
Many thanks @jeevatkm

**Comment by ppipada** [comment]
@jeevatkm Did you get any chance to identify the fix for issue seen? Any chance of getting the release soon?

**Reply by kishaningithub** [reply]
Also let know how the community can help..

**Comment by ppipada** [comment]
@jeevatkm Hello again. Anything we can do to speed  the release up? 
More point of concern are the data races. A released version with already incorporated fixes would help a lot in adoption.

Sorry for bothering.
